tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -50,11 +50,10 @@
 # @param options [Hash] Options hash literal to get configured VRF.
 # @result namespacestr [String] Returns 'sudo ip netns exec vrf <cmd>'
 # command string for 'cisco' platform.
-def get_namespace_cmd(host, cmdstr, options)
+def get_namespace_cmd(host, cmdstr, _options)
   case host['platform']
   when /cisco/
-    agentvrf = options[:HOSTS][host.to_s.to_sym]['vrf']
-    return "sudo ip netns exec #{agentvrf} " + cmdstr
+    return cmdstr
   else
     return cmdstr
   end
```

Comment:
Actually, bigger problems. We noticed a couple of things in the beaker code: beaker appears to be keying off `cisco-5` for nexus platforms and `cisco-7` for XR platforms:
   https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host/cisco.rb#L24
   https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host/cisco.rb#L75
   https://github.com/puppetlabs/beaker/blob/master/spec/beaker/host/cisco_spec.rb#L49

...which is a problem since those numbers are somewhat irrelevant to XR and somewhat interchangeable on the nexus side. After discussing with Glenn we think it would be better to use a more explicit platform pattern to differentiate these two platform families; ie: `cisco-nexus-` and `cisco-ios_xr-`.
 
I think we need to get this resolved before we can merge your PR.